src/Makefile.am

```diff
@@ -156,25 +156,20 @@ eos_updater_SOURCES = \
 
 dist_man8_MANS += docs/eos-updater.8
 
+eos_updater_dbus_if := com.endlessm.Updater
+
 dbusconfdir = $(datadir)/dbus-1/system.d
 dbussystemservicedir = $(datadir)/dbus-1/system-services
 
-dbusconf_DATA = $(eos_updater_dbus_if).conf
-dbussystemservice_DATA = $(eos_updater_dbus_if).service
+dbusconf_DATA = $(srcdir)/$(eos_updater_dbus_if).conf
+dbussystemservice_DATA = $(srcdir)/$(eos_updater_dbus_if).service
```

Comment:
Shouldn’t `dbusconf_DATA` and `dbussystemservice_DATA` both have a `dist_` prefix? Please double-check that `make distcheck` and building the Debian package works after applying these commits. (As I said on Slack, I use [`build-snapshot`](https://git.apertis.org/cgit/apertis-customizations.git/tree/development/build-snapshot) to do both.)